loom/engine/bindings/loom/lmApplicationTasks.cpp

```diff
@@ -33,8 +33,6 @@ extern "C"
 
 void loom_tick()
 {
-    LOOM_PROFILE_START(loom_tick);
```

Comment:
dump() should be called the next time the stack goes to zero - it should never be called directly. So that shouldn't happen and is a bug?